dev/bots/download_goldctl.ps1

```diff
@@ -8,7 +8,7 @@ $path = "C:\Windows\Temp\depot_tools"
 $gclient = "C:\Windows\Temp\depot_tools\gclient.bat"
 $cipd = "C:\Windows\Temp\depot_tools\cipd.bat"
 $ensureFile = "C:\Windows\Temp\depot_tools\ensure.txt"
-$text = "# Ensure File`n`$ServiceURL https://chrome-infra-packages.appspot.com`n`n# Skia Gold Client goldctl`nskia/tools/goldctl/`${platform} latest"
+$text = "# Ensure File`n`$ServiceURL https://chrome-infra-packages.appspot.com`n`n# Skia Gold Client goldctl`nskia/tools/goldctl/`${platform} git_revision:b57f561ad4ad624bd399b8b7b500aa1955276d41"
 
```

Comment:
Drive by suggestion: add docs that the revision refers to the chrome infra/infra repo, e.g. https://chromium.googlesource.com/infra/infra/+/b57f561ad4ad624bd399b8b7b500aa1955276d41